kmcd-data.md

```diff
@@ -69,9 +69,11 @@ We model everything with arbitrary precision rationals, but use sort information
  // -----------------------------------------
     rule Rad2Wad(FInt(R, RAD)) => FInt(R /Int RAY, WAD)
 
-    syntax Ray ::= Wad2Ray ( Wad ) [function]
+    syntax Ray ::= Wad2Ray ( Wad )      [function]
+                 | Wad2Ray ( MaybeWad ) [function]
```

Comment:
What do we need this `MaybeWad` variant for?

Reply:
We don't, my bad, removing it